migrations/versions/732790d814bc_.py

```diff
@@ -0,0 +1,36 @@
+"""empty message
+
+Revision ID: 732790d814bc
+Revises: 2f4db5864a2d
+Create Date: 2022-06-02 11:42:01.642873
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '732790d814bc'
+down_revision = '2f4db5864a2d'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
```

Comment:
This migration shouldn’t be in here. I’ve already done it

Reply:
I think I had to make it because of a head conflict. Can close and reopen without the migration.